src/app/integrations/create-page/flow-view/flow-view.component.ts

```diff
@@ -57,6 +68,18 @@ export class FlowViewComponent implements OnInit, OnDestroy {
     }
   }
 
+  getSteps() {
```

Comment:
ROFL, it's a little crazy.  Actually I think I can rework this a bit, it's to cater for the empty initial state of an integration but I don't love this solution really.